src/Inkluzitron/Modules/Vote/EndOfVotingScheduledTaskHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Discord;
+using Discord.WebSocket;
+using Inkluzitron.Contracts;
+using Inkluzitron.Data.Entities;
+using Inkluzitron.Enums;
+using Inkluzitron.Models.Vote;
+using System;
+using System.Threading.Tasks;
+
+namespace Inkluzitron.Modules.Vote
+{
+    public class EndOfVotingScheduledTaskHandler : IScheduledTaskHandler
+    {
+        private VoteService VoteService { get; }
+        private DiscordSocketClient Client { get; }
+
+        public EndOfVotingScheduledTaskHandler(VoteService voteService, DiscordSocketClient client)
+        {
+            VoteService = voteService;
+            Client = client;
+        }
+
+        public async Task<ScheduledTaskResult> HandleAsync(ScheduledTask scheduledTask)
+        {
+            if (scheduledTask.Discriminator != "EndOfVoting")
```

Comment:
Spíš než tenhle IF, tak bych spíš místo interface udělal abstract class, kde součástí base class by bylo, na co se to má chytat.

Reply:
Takhle mi to přijde flexibilnější a čitelnější, než když by se do toho zatahovala povinná abstraktní třída jako předek. Fakt se vyplatí mít kvůli jednomu ifu dědičnost?

Reply:
Ještě jsem přemýšlel nad attributem, ale to mi přišlo moc roztahané. Abstract class by to pokryla jak jako interface, tak jako discriminator.

Reply:
Nechme to tak